Mathlib/Topology/ContinuousFunction/Compact.lean

```diff
@@ -41,6 +41,17 @@ namespace ContinuousMap
 variable {α β E : Type*} [TopologicalSpace α] [CompactSpace α] [MetricSpace β]
   [NormedAddCommGroup E]
 
+section Support
+
+variable {F γ : Type*} [Zero γ] [TopologicalSpace γ] [FunLike F α γ]
+
+/-- In a compact space `α`, any continuous function has compact support. -/
+theorem isCompact_tsupport_of_CompactSpace (f : F) :
```

Comment:
The theorem name should match the statement. So it should be in terms of `HasCompactSupport`. Also, by the naming convention words do start with a lowercase letter. So
```suggestion
theorem hasCompactSupport_of_compactSpace (f : F) :
```
But the statement as written is not good, because you can see that it does not really involve `f`, only its coercion to functions. So there is no gain in having a `FunLike` type here, compared to just stating the result for functions.

Also, for now it's in the namespace `ContinuousMap`, while there is no `ContinuousMap` anywhere around in the statement. Better move it outside of this namespace, and probably to another file (`Mathlib.Topology.Support`, maybe?).

Reply:
OK, I will move this to `Mathlib.Topology.Support`, changing the name and for `f : α → γ` and pulling it outside the namespace.